healthcareai/supervised_model_trainer.py

```diff
@@ -26,19 +26,19 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
         self.grain_column = grain_column
 
         # Build the pipeline
-        # TODO This pipeline may drop nulls in prediction rows if impute=False
-        # TODO See https://github.com/HealthCatalyst/healthcareai-py/issues/276
         pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=impute)
+        prediction_pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=True)
 
         # Run the raw data through the data preparation pipeline
         clean_dataframe = pipeline.fit_transform(dataframe)
+        _ = prediction_pipeline.fit_transform(dataframe)
```

Comment:
This makes sense to me and I agree with forcing imputation on prediction, as long as it is documented. Would you mind adding a small bit of documentation in the markdown files (and maybe a small note here in the code?)